code/modules/clothing/suits/ego_gear/teth.dm

```diff
@@ -48,7 +48,7 @@
 	name = "life for a daredevil"
 	desc = "The fear of death is natural, but in denying it you shall find untold strength."
 	icon_state = "daredevil"
-	armor = list(RED_DAMAGE = 40, WHITE_DAMAGE = 25, BLACK_DAMAGE = 25, PALE_DAMAGE = -100) // -10
+	armor = list(RED_DAMAGE = 30, WHITE_DAMAGE = 20, BLACK_DAMAGE = 20, PALE_DAMAGE = -50) // -30
```

Comment:
I won't lie to you. I will forever mourn this day. Not for the loss of the damages, but for the loss of -50 Pale resist.